README.md

```diff
@@ -9,11 +9,15 @@
+**Inifinite Scale is currently in code freeze for general availability of version 2.0. Feature completeness was reached and the community is concentrating on bug fixes and performance improvements.** 
 
-The single binary allows scaling oCIS from a [Raspberry Pi](https://owncloud.dev/ocis/deployment/basic-remote-setup/) to [Kubernetes cluster](https://owncloud.dev/ocis/deployment/kubernetes/) by [changing the configuration and starting multiple services as needed](https://owncloud.dev/ocis/deployment/). The multiservice architecture allows tailoring the functionality to your needs and reusing services that may already be in place like [Keycloak](https://www.keycloak.org/) as the identity provider.
+Make sure to download the [latest release candidate build](https://download.owncloud.com/ocis/ocis/testing/) today!
+
+Infinte Scale allows [web](https://github.com/owncloud/web), [Android](https://github.com/owncloud/android), [iOS](https://github.com/owncloud/ios-app) and [desktop](https://github.com/owncloud/client/) clients to synchronize and share file spaces with a scalable server side based on [reva](https://reva.link/) using open and well defined APIs like [WebDAV](http://www.webdav.org/) and [CS3](https://github.com/cs3org/cs3apis/). External applications like [Collabora Online](https://github.com/CollaboraOnline/online), [OnlyOffice Docs](https://github.com/ONLYOFFICE/DocumentServer) or [Microsoft Office Online Server](https://owncloud.com/microsoft-office-online-integration-with-wopi/) can be used to collaborate using a [WOPI application gateway](https://github.com/cs3org/wopiserver). Users are authenticated using [OpenID Connect](https://openid.net/connect/) and the embedded [LibreGraph Connect](https://github.com/libregraph/lico) identity provider. 
+
+With a focus on easy install and operation, Infinite Scale is delivered as a single binary that allows scaling oCIS from a [Raspberry Pi](https://owncloud.dev/ocis/deployment/basic-remote-setup/) to [Kubernetes cluster](https://owncloud.dev/ocis/deployment/kubernetes/) by [changing the configuration and starting multiple services as needed](https://owncloud.dev/ocis/deployment/). The multiservice architecture allows tailoring the functionality to your needs and reusing services that may already be in place like [Keycloak](https://www.keycloak.org/) as the identity provider.
```

Comment:
@dragotin Q: why do we have on top links to dev docs whereas we have in the section below (Run ownCloud Infinite Scale) we correctly reference the admin docs. We should either use the same links (preferred) here or rephrase telling to point to dev docs for additional dev info. Else this gets confusing to readers. The admin docs have very detaild and actual/maintained info about these topics.

Reply:
Agreed, it should be clear that the admin docs should be the first source of truth for admins, and the dev docs only provide additional/detail info.
@mmattel could you provide a fix for this - just push into this branch?